_src/pldoc/static/js/tabs.js

```diff
@@ -0,0 +1,54 @@
+define([
+    'jquery'
+    ], function($) {
+
+    var Tabs = {
+
+        vars: {
+            tabContainer:   $('.pldoc-tab-wrapper'),
+            tabs:           $('.pldoc-tab-labels'),
+            tab:            $('.pldoc-tab-label'),
+            panels:         $('.pldoc-tabs'),
+            panel:          $('.pldoc-tab'),
+            activeClass:    'is-active',
+            hiddenClass:    'is-hidden',
+        },
+
+        init: function() {
+            Tabs.handleTabClick();
+        },
+
+        resetInterface: function(wrapper) {
+            wrapper.find('.pldoc-link').each(function(event) {
+                $(this).removeClass(Tabs.vars.activeClass);
+            });
+
+            wrapper.find(Tabs.vars.panel).each(function() {
+                $(this).removeClass(Tabs.vars.activeClass)
+                       .addClass(Tabs.vars.hiddenClass);
+            });
+        },
+
+        handleTabClick: function() {
+            var el, content;
```

Comment:
these vars are only used within the .on function so their declaration should be moved there.